src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IVariableDeclaration.vb

```diff
@@ -1,4 +1,4 @@
-' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
@333fred: looks like the BOM issues are back.  Please investigate

Reply:
@heejaechang this was fixed. Have you updated?

Reply:
@TyOverby @333fred I didn't use the tool here. I manually updated the file and file was on wrong encoding and VS fixed it by moving to UTF-8 (it was saved in western english something encoding)